jupyter_notebooks/pylibCZIrw/pylibCZIrw_3_3_0.ipynb

```diff
@@ -1265,7 +1266,7 @@
  "metadata": {
   "celltoolbar": "Raw Cell Format",
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "cziread39",
```

Comment:
Change not needed.
```suggestion
   "display_name": "Python 3 (ipykernel)",
```

Reply:
hmmm, but the base env Python 3 will usually never work. And it does not matter anyway, because every user will adapt this